app/assets/javascripts/admin/views/broken-links-report.js

```diff
@@ -0,0 +1,60 @@
+window.GOVUK = window.GOVUK || {}
+window.GOVUK.Modules = window.GOVUK.Modules || {};
+
+(function (Modules) {
+  function BrokenLinksReport (module) {
+    this.module = module
+  }
+
+  BrokenLinksReport.prototype.init = function () {
+    if (!fetch) return
+    var button = this.module.querySelector('button')
+    var authenticityTokenField = this.module.querySelector('[name="authenticity_token"]')
+    if (button && authenticityTokenField) this.setupEventListener(button, authenticityTokenField.value)
+    var refreshLink = this.module.querySelector('.js-broken-links-refresh')
+    if (refreshLink) {
+      this.setupPolling(refreshLink.href)
+      refreshLink.remove()
+    }
+  }
+
+  BrokenLinksReport.prototype.setupEventListener = function (button, authenticityToken) {
+    var action = this.module.querySelector('form').action
```

Comment:
Should we target a class rather than a `form` tag? for example `js-app-view-broken-links__refresh-form`

My reasoning behind this would primarily be that we may introduce another form or change the form element to something else. Having an explicit JS reference class will help developers be wary that a JS listener is attached this.

Reply:
Added 👍